hi-ml-cpath/Makefile

```diff
@@ -29,6 +29,10 @@ pip_from_conda:
 	sed -e '1,/pip:/ d' environment.yml | grep -v "#" | cut -d "-" -f 2- > temp_requirements.txt
 	pip install -r temp_requirements.txt
 
+# Lock the current Conda environment secondary dependencies versions
+lock_env:
+	./create_and_lock_environment.sh
```

Comment:
This script has been moved to root, any reason why?

Reply:
It's been moved because we use it in multiple subfolders.